lib/src/view/watch/tv_screen.dart

```diff
@@ -56,9 +57,6 @@ class _TvScreenState extends ConsumerState<TvScreen> {
     userId: widget.user?.id,
   );
 
-  final _whiteClockKey = GlobalKey(debugLabel: 'whiteClockOnTvScreen');
```

Comment:
Sorry for the delay.
I still cannot accept the PR because these keys are removed where they should be kept.